Extensions/TextInput/JsExtension.js

```diff
@@ -200,6 +209,30 @@ module.exports = {
         .setLabel(_('Width'))
         .setGroup(_('Border appearance'));
 
+      objectProperties
+        .getOrCreate('padding')
+        .setValue((objectContent.padding || 0).toString())
+        .setType('number')
+        .setLabel(_('Padding'))
+        .setGroup(_('Font'));
+      objectProperties
+        .getOrCreate('maxLength')
+        .setValue((objectContent.maxLength || 0).toString())
+        .setType('number')
+        .setLabel(_('Max length'))
+        .setDescription(_('The maximum length of the input value.'))
```

Comment:
I added a comment on the PR, I think you should add in the description that this property will not be used if the input type is a number.